pyppeteer/page.py

```diff
@@ -1168,8 +1162,7 @@ def viewport(self) -> Optional[Dict]:
         """
         return self._viewport
 
-    async def evaluate(self, pageFunction: str, *args: Any,
-                       force_expr: bool = False) -> Any:
+    async def evaluate(self, pageFunction: str, *args) -> Any:
```

Comment:
I assume puppeteer removed the force_expr when their detection algorithmbecame mature?

Reply:
I can't find any references to such parameter in puppeteer :grimacing:

Reply:
Yeah, it likely was due to miyakogi not fully trusting his detection algorithm to detect when a str is a function or expression